cli/js/web/blob.ts

```diff
@@ -124,6 +126,36 @@ function processBlobParts(
   return bytes;
 }
 
+function getStream(blobBytes: Uint8Array): ReadableStream {
+  return new ReadableStream({
+
```

Comment:
```
 ReadableStream({
  start(controller) {
      controller.enqueue(blobBytes);
```
Maybe?

I think these need some tests too. These would be great to have!

Reply:
ye added a test for the `text()` method. i was considering other tests as well, but text() relies on code that appears in the `arrayBuffer()` method, and that again uses code from the `stream()` method, so if any of them fail, the test would fail as well + testing those 2 would be a bit of a pain and stupid, as testing `arrayBuffer` would literally be the same code as `text()`